src/hooks/users/useWorldUsers.tsx

```diff
@@ -46,24 +49,37 @@ export const WorldUsersProvider: React.FC<WorldUsersProviderProps> = ({
     isDefined(sovereignVenueId) &&
     isDefined(venueId);
 
-  // TODO: refactor this to not use useFirestoreConnect
-  useFirestoreConnect(() => {
-    if (!shouldConnect) return [];
-
-    const relatedLocationIds = [venueId];
+  // // TODO: refactor this to not use useFirestoreConnect
+  // useFirestoreConnect(() => {
+  //   if (!shouldConnect) return [];
+  //
+  //   const relatedLocationIds = [venueId];
+  //
+  //   if (sovereignVenueId) {
+  //     relatedLocationIds.push(sovereignVenueId);
+  //   }
+  //
+  //   return [
+  //     {
+  //       collection: "users",
+  //       where: ["enteredVenueIds", "array-contains-any", relatedLocationIds],
+  //       storeAs: "worldUsers",
+  //     },
+  //   ];
+  // });
+
+  const worldUsersQueryResult = useWorldUsersQuery(
+    shouldConnect && venueId ? { relatedLocationIds: [venueId] } : skipToken
+    // { selectFromResult: (result) => result?.data?.userIds }
+  );
```

Comment:
[note] While this is obviously fairly placeholder/MVP for now.. this is basically how we can connect to the `worldUsersApi` data we defined earlier in `src/store/api/worldUsers.ts`